src/routes/(infoPages)/land/fire/safety/fire-danger-map/+page.server.ts

```diff
@@ -1,47 +1,69 @@
 import gql from "graphql-tag";
 import { print as printQuery } from "graphql";
+import { error } from "@sveltejs/kit";
 import parishTopoJSON from "./parishes.json";
 
-import type { FireWeatherMapQuery } from "./$queries.generated";
+import type { FireWeatherMapPageQuery } from "./$queries.generated";
 
 const query = gql`
-  query FireWeatherMap {
-    fireDangerMapCollection(limit: 1) {
+  query FireWeatherMapPage($metadataID: String!, $preview: Boolean = false) {
+    fireDangerMapCollection(
+      where: { pageMetadata: { sys: { id: $metadataID } } }
+      limit: 1
+      preview: $preview
+    ) {
       items {
+        title
         description {
           json
         }
+        pageMetadata {
+          sys {
+            id
+          }
+        }
```

Comment:
Besides making sure we grab the appropriate entry, I also added the `title` so that we could use whatever the editors provide in Contentful.

Reply:
Ah okay, yeah I didn't see that help text (but I also might have missed it) so I figured it would be more confusing for the title to not be in-use.